### DISCUSSION TITLE
createLocalizedPathnamesNavigation with next build failed to start

### DISCUSSION THREAD

**Discussion by WenLonG12345** [amannn/next-intl#1247] [Q&A]
Hi, I am using this library with `createLocalizedPathnamesNavigation` and here is the snippets of my `i18n.ts` file

```
export const pathnames = {
  '/video': getMappedPath(process.env.VARIANT, '/video'),
  '/video/[videoGroupId]': getMappedPath(
    process.env.VARIANT,
    '/video/[videoGroupId]',
  ),
  '/manage-feed': getMappedPath(process.env.VARIANT, '/manage-feed'),
  '/my-feed': getMappedPath(process.env.VARIANT, '/my-feed'),
  '/latest-news/[id]': getMappedPath(process.env.VARIANT, '/latest-news/[id]'),
};

export const { Link, redirect, usePathname, useRouter, getPathname } =
  createLocalizedPathnamesNavigation({ locales, localePrefix, pathnames });

```

I am using `next build` to build my application and served it with `next start` . However, it throws the following error 
![image](https://github.com/user-attachments/assets/00706a53-2360-4cf1-ba26-e53ea99fe1d5)

and I go deeper, it is actually caused by the localized `pathnames`. Is there any idea how to solve this?

**Comment by amannn** [comment]
Have you tried logging your `pathnames` object? Maybe you can spot a configuration issue there.
